src/style.css

```diff
@@ -7,7 +7,7 @@
 }
 
 body {
-  background-image: linear-gradient(rgba(0, 0, 0, 0.6), rgba(0, 0, 0, 0.6)), url(./image.jpg);
```

Comment:
- [ ] It will be better if you uncommit this line so that your background would be more better